zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -754,7 +768,11 @@ def detect_tensorfusion_capability(cls, pci_device_to):
             logger.debug('TensorFusion capability check skipped for %s: SR-IOV VF is not eligible' % addr)
             return False, {}
 
-        r, o, e = bash_roe('nvidia-smi --query-gpu=pci.bus_id,driver_version --format=csv,noheader -i %s' % addr)
+        if cls._is_bound_to_vfio(addr):
+            logger.debug('TensorFusion capability check skipped for %s: device bound to vfio-pci (passthrough)' % addr)
+            return False, {}
+
+        r, o, e = bash_roe('timeout 10 nvidia-smi --query-gpu=pci.bus_id,driver_version --format=csv,noheader -i %s' % addr)
```

Ruff (0.15.10)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)